config.toml.example

```diff
@@ -182,6 +182,10 @@
 # essentially skipping stage0 as the local compiler is recompiling itself again.
 #local-rebuild = false
 
+# Print out how long each rustbuild step took in JSON format (mostly intended
```

Comment:
Those `println!` output doesn't look like JSON at all 😄

Reply:
This also prints both how long rustc takes to compile each crate and how long each step takes; I'd sort of expect to see this documented here?

It's also a bit odd that rustc prints to stderr whereas the step timing is printed to stdout.